client/src/styles/userforms.scss

```diff
@@ -30,7 +30,7 @@
   .step-button-jump {
     position: absolute;
     top: 0;
-    opacity: .7;
+    opacity: 0.7;
```

Comment:
I believe eslint asks you _not_ to use leading zeroes

Reply:
It was added by my linter, but eslint didn't complain on build.
I'll change it back though :>